<launch>
 <arg name="mav_name" default="vfly"/>
 <arg name="type" default="x"/>
  <group ns="$(arg mav_name)">
  <node name="omni_adrc_controller_node" pkg="vfly" type="omni_adrc_controller_node"  output="screen">
      <rosparam command="load" file="$(find vfly)/config/$(arg mav_name)_omni_adrc_controller.yaml" />
      <rosparam command="load" file="$(find vfly)/config/$(arg mav_name)_vehicle.yaml" />
      <param name="vehicle/type" type="str" value="$(arg type)" />
  </node>
  </group>
</launch>